common/metrics.py

```
@@ -0,0 +1,209 @@
+        """
+        if cache_type == "hit":
+            cls.cached_queryset_hit.labels(model=model).inc()
+        elif cache_type == "miss":
+            cls.cached_queryset_miss.labels(model=model).inc()
+        elif cache_type == "eviction":
+            cls.cached_queryset_evicted.labels(model=model).inc()
+        elif cache_type == "invalidated":
+            cls.cached_queryset_invalidated.labels(model=model, reason=reason or "unknown").inc()
+
+    @classmethod
+    def update_cache_stats(cls, model: str, stats: Dict[str, Any]) -> None:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)